src/plugins/comments/test/comments.e2e.js

```diff
@@ -232,6 +232,22 @@ describe('Comments', () => {
       expect(afterSetCellMetaCallback).toHaveBeenCalledWith(1, 1, 'comment', { value: 'Added comment' }, undefined, undefined);
     });
 
+    it('should not trigger `afterSetCellMeta` callback if `setCommentAtCell` function is invoked and `beforeSetCellMeta` returned false', () => {
+      const afterSetCellMetaCallback = jasmine.createSpy('afterSetCellMetaCallback');
+      const hot = handsontable({
+        data: Handsontable.helper.createSpreadsheetData(4, 4),
+        comments: true,
+        beforeSetCellMeta: () => false,
+        afterSetCellMeta: afterSetCellMetaCallback
+      });
+
+      const plugin = hot.getPlugin('comments');
+
+      plugin.setCommentAtCell(1, 1, 'Added comment');
+
+      expect(afterSetCellMetaCallback).not.toHaveBeenCalled();
+    });
+
```

Comment:
This test should only test that the `setCellMeta` API is used internally by the Columns plugin (by spying the method, or checking visually that the comment was not rendered). Nothing more. 

I've seen that you have written the same tests for the other plugins. They are unnecessary because the tests you've been written here https://github.com/handsontable/handsontable/pull/6778/files#diff-480d01009506d13fce635808fa325351R112 are proved already that it works. The repetition is redundant.